cloudfour/cloudfour.com-patterns
# Add variable fonts

Closes #469 

https://deploy-preview-506--cloudfour-patterns.netlify.com/?path=/story/demo-button--button-element

## Testing

- Apply different font weights and styles to the button using devtools to verify that the Source Sans Pro variable font is working correctly
- Change the button to use "Source Code Pro" and verify that it works (this is not a variable font)

Test in IE11 to see if the Source Sans Pro non-variable font fallbacks are loaded
